### PR TITLE
Add etcd procedures repo

### PR DESCRIPTION
**Pull request by makentenza**
#### What is this PR About?
Resources to operate an etcd Cluster for Openshift. This repository will contain different documents and playbooks to operate etcd. This PR is covering the first procedure, scale up an etcd Cluster. A document with detailed steps is provided and a playbook is included as well in order to automate the process.

#### How do we test this?
Manual steps: Follow instructions from docs/README.md
Automatic playbook: Follow instructions from playbooks/README.md

cc: @redhat-cop/day-in-the-life-ops

### CODE REVIEW

File: etcd-procedures/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Different procedures to operate etcd in OCP
+
+## Project Description
+
+The aim of this repository is to generate the required tools (ansible playbooks and/or automated scripts) and documentation to operate an etcd cluster for **Openshift Container Platform** and cover the following topics
+
+* Add new member to an existing Cluster
```
Comment (sabre1041) [inline etcd-procedures/README.md]: Add links to sub pages as necessary

File: etcd-procedures/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## How to use these documents
+
+The following documents can be used to operate an etcd Cluster. Each one contains detailed explanation, commands included step by step, about a particular procedure for an etcd Cluster.
+
+## Documents
+
+* **add_etcd_node.adoc**: this document can be used to **scale up** an existing etcd Cluster with new member(s). This is only aplicable to containerized etcd Clusters.
```
Comment (sabre1041) [inline etcd-procedures/docs/README.md]: Add link to `add_etcd_node.adoc`

File: etcd-procedures/docs/add_etcd_node.adoc [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- DNS working
+- Access to registry.access.redhat.com for image pulling
+- SSH access from the existing etcd Nodes to new etcd Nodes
+
+
+=== Install etcd as a Docker container
+
+NOTE: Check if etcd needs to be installed in order to perform OCP upgrades
+
+==== From one of the existing etcd Nodes, get the etcd container image version we need to run
```
Comment (sabre1041) [inline etcd-procedures/docs/add_etcd_node.adoc:20]: Would you be able to provide context into what the commands that are performing?

File: etcd-procedures/playbooks/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+## How to use these playbooks
+
+In order to use these playbooks, an updated Ansible Inventory file for your OCP Cluster is needed. This Inventory file must follow the instructions provided during the [Advance Install for OCP](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-ansible)
+
+These playbooks will identify the etcd members to operate by adding the following to the Inventory file:
+
+* A new entry called **new_etcd** under [OSEv3:children] group
```
Comment (sabre1041) [inline etcd-procedures/playbooks/README.md]: Would a sample inventory be more helpful?

File: etcd-procedures/playbooks/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+* A new group called **[new_etcd]**
+
+## Playbooks
+
+* **etcd_scaleup.yml**: automatically scales up an existing etcd cluster with a new member included in [new_etcd] group. This is only aplicable to containerized etcd Clusters.
```
Comment (sabre1041) [inline etcd-procedures/playbooks/README.md]: Add link

**Review by sabre1041 — CHANGES_REQUESTED** [review]
@makentenza Performed a review of the documentation and not yet of the steps/playbooks.

I really like this as it is really needed. Since there is mixed document format (markdown vs asciidoc), it might make sense that the entire repository migrate to asciidoc. thoughts?

File: etcd-procedures/docs/add_etcd_node.adoc (hunk 3 above)
Comment (makentenza) [reply etcd-procedures/docs/add_etcd_node.adoc:20]: Do you mean the below commands?

Containerized etcd:
      # docker ps | grep etcd | awk {'print $2'}
Package etcd:
      # etcd --version | grep etcd | awk {'print "registry.access.redhat.com/rhel7/etcd:"$3'}

**Comment by makentenza** [comment]
I'm closing this PR as we are moving this to openshift-toolkit as discussed

https://github.com/redhat-cop/openshift-toolkit/pull/7

**Pull request closed without merging**